apps/cowswap-frontend/src/legacy/components/AddressInputPanel/index.tsx

```diff
@@ -8,10 +9,12 @@ import { UI } from '@cowprotocol/ui'
 import { useWalletInfo } from '@cowprotocol/wallet'
 
 import { t, Trans } from '@lingui/macro'
+import { getAddress } from 'ethers/lib/utils'
```

Comment:
Please, use `import { getAddress } from '@ethersproject/address'` instead

Reply:
Ok. When are you guys moving to ethers v6?

Reply:
Good question! we have a dependency with the contracts NPM package, so either when we migrate that or when we kill that dependency. Not urging for now.

Alternatively never, if we move to viem :P

Reply:
happy to help if migration to viem is considered! :D